meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,134 @@
+# v0.29.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using this version in production. This is only for test purposes. Please, feel free to [report bugs](https://github.com/meilisearch/meilisearch/issues/new/choose) and [give feedback about the new feature introductions](https://github.com/meilisearch/product/discussions).
+
+In this release, we integrated a demanded feature: requiring all query terms to be present in search results. Some users have been stuck in the pre-0.21 versions because of this.
+We also improved the indexing speed
+
+🧰 Most integrations (SDKs, clients, libraries) will be compatible with this version within four hours of the release. Sometimes it can take longer, maybe 24 hours, depending on the issues we can encounter during the release.
+However, here is the exhaustive list of **non-compatible** integrations with v0.29.0:
+- [meilisearch-dart](https://github.com/meilisearch/meilisearch-dart) and [meilisearch-swift](https://github.com/meilisearch/meilisearch-swift): You can use it with v0.29.0 but the [`matchingStrategy` feature](#customize-matching-strategy) is not integrated yet. Let us know if you really need it, and contributions are always welcome!
+- [meilisearch-java](https://github.com/meilisearch/meilisearch-java): Our team is still working on refactoring the code, and the SDK cannot be used with v0.29.0. Follow the repository to know when it will be compatible.
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 The official documentation is not updated yet, but will be ready for the official v0.29.0 release. The work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Customize matching strategy
+
+It's now possible to control how Meilisearch matches documents during search by choosing to return only documents containing **all** the query words.
+
+### How to activate it?
+
+```bash
+curl \
+  -X POST 'http://localhost:7700/indexes/movies/search' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{ "q": "american ninja", "matchingStrategy": "all" }'
+```
+
+Here Meilisearch will only return documents containing "american" AND "ninja".
+
+By default, Meilisearch will still return documents containing all the query words, followed by documents that don't contain all the query words.
```

Comment:
```suggestion
Meilisearch's default matching strategy is `last`. It will still return documents containing all the query words, followed by documents that don't contain all the query words.
```